cdpctl/validation/aws_utils.py

```diff
@@ -63,32 +71,25 @@ def get_client(client_type: str, config):
         config,
         "infra:aws:profile",
         key_value_expected=False,
-        key_missing_message=(
-            "No profile has been defined for config option: env:aws:profile"
-        ),
-        data_expected_error_message=(
-            "No profile was provided for config option: env:aws:profile"
-        ),
+        key_not_found_issue=AWS_PROFILE_CONFIG_NOT_DEFINED,
+        data_expected_issue=AWS_PROFILE_NOT_DEFINED,
     )
     region_name: Optional[str] = get_config_value(
         config,
         "infra:aws:region",
         key_value_expected=False,
-        key_missing_message=(
-            "No region has been defined for config option: env:aws:region"
-        ),
-        data_expected_error_message=(
-            "No region was provided for config option: env:aws:region"
-        ),
+        key_not_found_issue=AWS_REGION_CONFIG_NOT_DEFINED,
+        data_expected_issue=AWS_REGION_NOT_DEFINED,
     )
 
-    if profile_name:
-        session = boto3.session.Session(profile_name=profile_name)
-        return session.client(client_type)
     if region_name:
+        if profile_name:
```

Comment:
could this be combined with line above?

Reply:
No we still want the return for the region without a profile name.  The region has to be defined.  Region and profile CAN be defined and returns a different client.